mypyc/test-data/run-classes.test

```diff
@@ -2783,21 +2785,34 @@ def test_buffer_int_size() -> None:
         assert len(b.getvalue()) == 1
         b = Buffer(b.getvalue())
         assert read_int(b) == i
-    for i in (-12345, -12344, -11, 118, 12344, 12345):
+    for i in (-100, -11, 118, 12344, 16283):
```

Comment:
Test also each power of two up to, say, 200, just in case (positive and negative)? Maybe test both `n**2` (lower bits all zero) and  `n**2 - 1` (lower bits all ones).